# Petición (puede estar vacía, ya que la acción siempre es la misma)
---
# Respuesta
bool exito
string mensaje
